modules/feature-pack/src/main/resources/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-client/main/module.xml

```diff
@@ -20,31 +21,32 @@
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
-<module name="org.jboss.ws.cxf.jbossws-cxf-client" xmlns="urn:jboss:module:1.9">
+
+<module xmlns="urn:jboss:module:1.9" name="org.jboss.ws.cxf.jbossws-cxf-client">
 
     <dependencies>
         <!-- JAXB API + REF IMPL -->
         <module name="javax.xml.bind.api" export="true"/>
-        <module name="com.sun.xml.bind" export="true" services="export"/>
+        <module name="org.glassfish.jaxb" export="true" services="export"/>
         <module name="javax.xml.ws.api" export="true"/>
         <module name="javax.jws.api" export="true"/>
         <!-- WSDL4J API -->
         <module name="javax.wsdl4j.api" export="true"/>
         <!-- JBossWS API -->
-        <module name="org.jboss.ws.api" export="true"/>
+        <module name="org.jboss.ws.api" export="true" />
```

Comment:
this is just a formatting issue, but some changed lines add this space before closing tag while the rest is kept as is. It isn't a functional issue but it doesn't look good to have the mix of two formats

Reply:
this applies for all module XML files in this PR.

Reply:
I already formatted all items.

Reply:
There are still some inconsistencies, see:
```
grep -r " />" --include=module.xml
```

- https://github.com/jimma/jbossws-cxf/blob/galleon/modules/feature-pack/src/main/resources/modules/system/layers/base/org/jboss/ws/common/main/module.xml#L42
- https://github.com/jimma/jbossws-cxf/blob/galleon/modules/feature-pack/src/main/resources/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-server/main/module.xml
- https://github.com/jimma/jbossws-cxf/blob/galleon/modules/feature-pack/src/main/resources/modules/system/layers/base/org/jboss/ws/jaxws-client/main/module.xml